src/sql/platform/accounts/common/accountPickerViewModel.ts

```diff
@@ -20,7 +20,7 @@ export class AccountPickerViewModel {
 	public selectedAccount: azdata.Account | undefined;
 
 	constructor(
-		private _providerId: string,
+		_providerId: string,
```

Comment:
any reason for making it public? if you do want to make it public, we should get rid of the underscore

Reply:
It's not making it public, it's making it not assigned to the local `this` scope

Reply:
Annoying TS/JS syntax!